LocationList.py

```diff
@@ -135,8 +135,8 @@ def shop_address(shop_id: int, shelf_id: int) -> int:
     ("KF Sarias House Recovery Heart 2",                             ("Freestanding", 0x29,  (0,0,4), None,                         'Recovery Heart',                        ("Kokiri Forest", "Forest Area", "Freestandings",))),
     ("KF Sarias House Recovery Heart 3",                             ("Freestanding", 0x29,  (0,0,5), None,                         'Recovery Heart',                        ("Kokiri Forest", "Forest Area", "Freestandings",))),
     ("KF Sarias House Recovery Heart 4",                             ("Freestanding", 0x29,  (0,0,6), None,                         'Recovery Heart',                        ("Kokiri Forest", "Forest Area", "Freestandings",))),
-    ("KF Shop Blue Rupee",                                           ("ActorOverride",0x2D,  0x01, ([0x02587098], [
-        0x00, 0x15, 0x00, 0x92, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x06            ]),               'Rupees (5)',                            ("Kokiri Forest", "Forest Area", "Freestandings",))),
+    #("KF Shop Blue Rupee",                                           ("ActorOverride",0x2D,  0x01, ([0x02587098], [
```

Comment:
So this was recategorized into `KF Shop Wonderitem`? You can just delete this line if you want then

Reply:
Done